app/app.go

```diff
@@ -580,11 +606,12 @@ func NewStrideApp(
 	// - transfer
```

Comment:
The naming of these stacks has gotten quite confusing as they've grown. I'm going to add a TODO for myself to try and clean these up